packages/mobile/src/screens/upload-screen/screens/SelectTrackScreen.tsx

```diff
@@ -53,7 +53,8 @@ export const SelectTrackScreen = () => {
     useAsyncFn(async () => {
       try {
         const trackFile = await DocumentPicker.pickSingle({
-          type: DocumentPicker.types.audio
+          type: DocumentPicker.types.audio,
+          copyTo: 'cachesDirectory'
```

Comment:
What is this for?

Reply:
yeah its an interesting case. without this, the file is saved in /tmp file, which was seemingly getting removed if you wait like 20 seconds before uploading (which is common when filling all the stuff out). so this is basically a more stable option.